src/server/CgiServer.php

```diff
@@ -85,7 +85,7 @@ protected function extractRequest()
             'method' => strtoupper($_SERVER['REQUEST_METHOD']),
             'path' => parse_url($requestUri, PHP_URL_PATH),
             'headers' => $this->extractHeaders(),
-            'params' => $_GET,
+            'queryString' => $_SERVER['QUERY_STRING'],
```

Comment:
`$_SERVER['QUERY_STRING']` may not defined?

Reply:
yes, it's possible. I will fix it.